11ty/definitions/-misia.md

```diff
@@ -3,6 +3,7 @@ title: -misia
 slug: -misia
 speech: noun
 defined: true
+socialPreview: from Greek for hate or hatred
```

Comment:
Added the `socialPreview` front matter to just a few existing definitions, for testing and review purposes. If we adopt this approach, I'll go back and add it to the others.